README.md

```diff
@@ -30,6 +30,23 @@ Prophet is currently only supported via pip installation in EvalML.
 ```shell
 pip install evalml[prophet]
 ```
+Another option for installing Prophet with CmdStan as a backend is to use `make installdeps-prophet`.
```

Comment:
You might want to think about moving this instructions to **Install page on the docs**. This currently will make the README.md much longer. You could link from the README.md to the Install documentation page.

We put graphviz install instructions in our Install docs:
https://featuretools.alteryx.com/en/latest/install.html#installing-graphviz

Reply:
@gsheni Thanks for the recommendation!

Reply:
I also agree with this recommendation @ParthivNaresh !